.env.prod.template

```
@@ -56,6 +56,7 @@ PYTHONPATH=.
 
 # Repository name
 GITHUB_REPO=tunetrees
+PRODUCTION_BASE_URL="https://tunetrees.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)